ansible/group_vars/all/vars.yml

```diff
@@ -47,3 +48,4 @@ passwd:
 ansible_venv: /root/venv
 docker_apt_ver: ""
 airflow_tag: "1.8.0-oo3"
+oomsm_tag: "1.0.0-beta.1"
```

Comment:
Why did you put the tag in here? I had to resort to grepping the tree in order to find where this was defined.

The convention in other roles is to put this inside of `defaults/main.yml` and if needed override it from the deploy playbook.